src/Item_Project.php

```diff
@@ -265,16 +269,97 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
 
     public static function displayTabContentForItem(CommonGLPI $item, $tabnum = 1, $withtemplate = 0)
     {
+        /** @var array $CFG_GLPI */
+        global $CFG_GLPI;
 
         switch ($item->getType()) {
             case 'Project':
                 self::showForProject($item);
                 break;
 
             default:
-               // Not defined and used now
-               // Project::showListForItem($item);
+                if (in_array($item->getType(), $CFG_GLPI["project_asset_types"])) {
```

Comment:
A right check should be done here too, in order to prevent information leaking if someone calls the tab rendering endpoint manually.

```suggestion
                if (
                    Session::haveRight("project", Project::READALL)
                    && $item instanceof CommonDBTM
                    && in_array($item->getType(), $CFG_GLPI["project_asset_types"])
                ) {
```